### DISCUSSION TITLE
How to adjust the TOTP time token period using an API

### DISCUSSION THREAD

**Discussion by BrijeshDMIsystems** [ant-media/Ant-Media-Server#6084] [Q&A]
How do we Adjust the TOTP time token period via an API for this function. Doing it via the AMS Web Panel is straightforward, but we're unable to find an API for this. Guidance towards relevant documentation would be greatly appreciated.

**Comment by USAMAWIZARD** [comment]
@lastpeony  as far as I know there is no direct way to specify the TOTP period in the APIs , if @BrijeshDMIsystems  still need this then they might need to update the application settings with the rest apis , But would that be a good approx @lastpeony  ?

**Comment by mekya** [comment]
Hi @BrijeshDMIsystems ,

Let me update you. There is a `timeTokenPeriod` field in AppSettings. You can update it on `App > Settings > Advance``
I hope it helps. 
Let me know if we can help you further. 
 
<img width="1976" alt="Screenshot 2024-05-06 at 20 16 39" src="https://github.com/ant-media/Ant-Media-Server/assets/3456251/8493d205-d371-432a-acec-89a2330497b1">

Cheers
Oguz
